docs/getting-started.md

```diff
@@ -23,11 +23,10 @@ sudo setfacl -m u:${USER}:rw /dev/kvm
 
 The generic requirements are explained below:
 
-- **Linux 4.14+**
+- **Linux**
 
   Firecracker currently supports physical Linux **x86_64** and **aarch64**
```

Comment:
I would put again the kernel versions explicitly, and *maybe* a reference to the support policy.